src/TodoList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import EmptyTodoList from './EmptyTodoList';
+import List from './List';
+import TodoItem from './TodoItem';
+
+const TodoList = ({ todos, deleteTodo, checkTodo }) => {
+  if (todos.length === 0) return <EmptyTodoList>할 일이 없어요!</EmptyTodoList>;
```

Comment:
```suggestion
  if (todos.length === 0) {
    return (
      <p>할 일이 없어요!</p>
    );
  }
```

컴포넌트를 분리한 것과 그냥 작성해준 것 사이에 차이가 있나요?

Reply:
다양하게 한번 시도해보려고 했던 욕심이였던 것 같습니다😂  피드백 감사합니다. 삭제했습니다!